core/src/test/java/org/elasticsearch/VersionTests.java

```diff
@@ -189,7 +189,17 @@ public void testAllVersionsMatchId() throws Exception {
                 logger.info("Checking " + v);
                 assertEquals("Version id " + field.getName() + " does not point to " + constantName, v, Version.fromId(versionId));
                 assertEquals("Version " + constantName + " does not have correct id", versionId, v.id);
-                assertEquals("V_" + v.number().replace('.', '_'), constantName);
+                if (v.major >= 2) {
```

Comment:
You could remove this conditional, since the replacements below will be noops for pre 2.0 versions?